ml4h/recipes.py

```diff
@@ -207,6 +217,123 @@ def inference_file_name(output_folder: str, id_: str) -> str:
     return os.path.join(output_folder, id_, 'inference_' + id_ + '.tsv')
 
 
+def evaluate_batch(args):
+    """Evaluates batch-wise information for a model given the metrics
+    it was compiled with. This temporary function works only with ML4H
+    TensorGenerators.
+    """
+    # Local includes only required in this case.
+    from tensorflow.keras.models import load_model
+    from ml4h.metrics import get_metric_dict
+    # Local includes only required in this case.
+    from ml4h.evaluate_segmentations import BatchMetricsLogger
+    import seaborn as sns
```

Comment:
move plotting to plots.py and maybe wait for subsequent PR to add plotting

Reply:
What are these plots showing?  What is Rank referring to?

Reply:
Sort order (rank)